### PR TITLE
PLT-6581 breakup webhooks greater than 4K into multiple posts

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: app/webhook.go [hunk 1]
```diff
@@ -159,11 +159,37 @@ func CreateWebhookPost(userId, teamId, channelId, text, overrideUsername, overri
...
+	splits = append(splits, remainingText)
+
+	var firstPost *model.Post = nil
+
+	for _, txt := range splits {
+		post.Id = ""
+		post.UpdateAt = 0
+		post.CreateAt = 0
+		post.Message = txt
+		if _, err := CreatePost(post, teamId, false); err != nil {
```
Comment (jwilander) [inline app/webhook.go]: Only worry is that this may introduce a race with `post` since we pass it to some goroutines down the line

Comment (coreyhulen) [inline app/webhook.go]: I had the same worry at first, but the post is saved synchronously with `<-Srv.Store.Post().Save(post);` in `CreatePost()`.

Comment (jwilander) [inline app/webhook.go]: Ok cool and anything after that is in a new post object from what the store returns

**Comment by jostyee** [comment]
I merged this PR but still not working.
screen shot 2017-06-14 at 12 40 07 pm

**Comment by jasonblais** [comment]
@jostyee this PR changed the behaviour for webhook messages only -- no change for user messages (yet)

**Comment by jostyee** [comment]
@jasonblais oops, just read the source diff and found that I misunderstood this PR, thanks for ur reply.

**Pull request merged**